interfaces/matlab/CMakeLists.txt

```diff
@@ -7,229 +7,371 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-# Generate SWIG wrapper for MATLAB
-if(HELICS_SWIG_GENERATE_INTERFACE_FILES_ONLY)
-    include(${CMAKE_CURRENT_SOURCE_DIR}/matlabSwigGenerateOnly.cmake)
-else()
-    find_package(Matlab COMPONENTS MAIN_PROGRAM)
+cmake_minimum_required(VERSION 3.4)
```

Comment:
don't we normally use 3.5?

Reply:
3.4 is what we use in most places. The benchmarks have a line setting it to 3.5.1 and package_config_tests set it to 3.6.0, everywhere else uses 3.4 (other than 3rd party).